lib/bitmap.rb

```diff
@@ -0,0 +1,87 @@
+class Bitmap
+  attr_reader :pixels, :width, :height
+
+  def initialize(width = 1, height = 1)
+    @width = width.to_i
+    @height = height.to_i
+    @pixels = Array.new(@height)
+    clear
+  end
+
+  def clear
+    @pixels.map! { |_row| WHITE * @width }
```

Comment:
These are `attr_reader`s so we don't need the `@`s (there's a few others dotted around as well)
```suggestion
    pixels.map! { |_row| WHITE * width }
```